Web/SFB.Web.UI/Controllers/SchoolController.cs

```diff
@@ -98,6 +98,12 @@ public class SchoolController : Controller
             ViewBag.ChartFormat = format;
             ViewBag.ShouldShowDashBoard = schoolVM.LatestYearFinancialData?.TotalExpenditure != null;
 
+            // AB#63763: use CFR instead of GIAS pupil count
+            if (schoolVM.LatestYearFinancialData?.PupilCount.GetValueOrDefault() > 0)
```

Comment:
Potentially could use pattern matching if C# version supports

Reply:
We're using C# 7.3 which does not, and I couldn't get this working by bumping to the maximum for .NET Framework apps, C# 8.